pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -157,7 +161,10 @@ spec:
     id: putcstorpooldeployment
   post: |
     {{- jsonpath .JsonResult "{.metadata.name}" | trim | addTo "putcstorpooldeployment.objectName" .TaskResult | noop -}}
-  task: |-
+    {{- jsonpath .JsonResult "{.metadata.uid}" | trim | addTo "putcstorpooldeployment.objectUID" .TaskResult | noop -}}
```

Comment:
How is the above `uid` used?  - should it be removed or ignored ?

Reply:
Its not getting used any where .... we remove it .